CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased][HEAD]
 
+* Bug Fixes
+   * **downloads**: fix broken search for previously cached download files with a format-name including whitespace. Keywords actually don't require inclusion of the export-format type, since the search already includes a "type" which refers to the export-format.
```

Comment:
you shouldn't need to modify the CHANGELOG if you run `npm run c`

Reply:
Ah, thanks.  I didn't know that!